automatic/virtualbox_5.0.x/virtualbox_5.0.x.nuspec

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>virtualbox</id>
+    <title>VirtualBox</title>
+    <version>0.0</version>
+    <authors>Oracle Corporation</authors>
+    <owners>chocolatey, Rob Reynolds, Mike Dellanoce</owners>
+    <summary>VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use.</summary>
+    <description>
+VirtualBox is a cross-platform virtualization application. It installs on existing Intel or AMD-based computers, whether they are running Windows, Mac, Linux or Solaris operating systems. It extends the capabilities of your existing computer so that it can run multiple operating systems (inside multiple virtual machines) at the same time.
+
+## Package parameters
+
+- `CurrentUser`       - Install for current user only
+- `NoDesktopShortcut` - Do not create desktop shortcut
+- `NoQuickLaunch`     - Do not create quick launch icon
+- `NoRegister`        - Do not register virtualbox file extensions
+- `NoPath`            - Do not add virtualbox install directory to the PATH
+- `NoExtensionPack`   - Do not install extension pack
+    </description>
+    <projectUrl>https://www.virtualbox.org/</projectUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/virtualbox</packageSourceUrl>
```

Comment:
I see I have the wrong package source url, I'll need to fix this when I get back.